app/src/main/java/protect/card_locker/LoyaltyCardViewActivity.java

```diff
@@ -704,7 +708,13 @@ protected void onResume() {
             AlertDialog.Builder builder = new MaterialAlertDialogBuilder(LoyaltyCardViewActivity.this);
             builder.setTitle(R.string.cardId);
             builder.setView(cardIdView);
-            builder.setPositiveButton(R.string.ok, (dialogInterface, i) -> dialogInterface.dismiss());
+            builder.setPositiveButton(R.string.copy_value, (dialog, which) -> {
```

Comment:
I'm not sure if this makes sense as the positive button, the default action probably shouldn't be to copy to clipboard.